private_sharing/migrations/0003_auto_20160909_0427.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.9 on 2016-09-09 04:27
+from __future__ import unicode_literals
+
+from string import digits  # pylint: disable=deprecated-module
+
+from django.db import migrations
+
+from common.utils import generate_id
+
+
+def migrate_go_viral_files(apps, *args):
+    Member = apps.get_model('open_humans', 'Member')
+    DataFile = apps.get_model('data_import', 'DataFile')
+    OnSiteDataRequestProject = apps.get_model('private_sharing',
+                                              'OnSiteDataRequestProject')
+    DataRequestProjectMember = apps.get_model('private_sharing',
+                                              'DataRequestProjectMember')
+    ProjectDataFile = apps.get_model('private_sharing', 'ProjectDataFile')
+    UserData = apps.get_model('go_viral', 'UserData')
+
+    try:
+        rumi = Member.objects.get(user__username='rumichunara')
+    except Member.DoesNotExist:
+        return
+
+    def random_id():
+        code = generate_id(size=8, chars=digits)
+
+        while DataRequestProjectMember.objects.filter(
+                project_member_id=code).count() > 0:
+            code = generate_id(size=8, chars=digits)
+
+        return code
+
+    project = OnSiteDataRequestProject(
+        is_study=True,
+        name='GoViral (2014-2016)',
+        slug='go-viral-archive',
+        leader='Dr. Rumi Chunara',
+        organization='NYU Polytechnic School of Engineering',
+        is_academic_or_nonprofit=True,
+        contact_email='rumi.chunara@nyu.edu',
+        info_url='https://www.goviralstudy.com/',
+        short_description="""Participants in this viral surveillance study can
```

Comment:
Short: Participants in this viral surveillance study received kits, and sent samples if they got sick.

Long: This project represents previous years of the GoViral study. Participants in this viral surveillance study received kits, and sent samples if they got sick. Data import from this study is no longer available, but data remains in member accounts if it was previously imported.

(Of course, we can change these later via admin panel too...)